src/bank00.asm

```diff
@@ -9006,9 +9006,24 @@ getDialogTextInsertionPoint:
     ld   A, [wWindowTextSpaceLeftOnLine]               ;; 00:375f $fa $ba $d8
     ld   C, A                                          ;; 00:3762 $4f
     ret                                                ;; 00:3763 $c9
-    db   $d5, $3e, $7f, $15, $cd, $44, $38, $14        ;; 00:3764 ????????
-    db   $3e, $7f, $cd, $44, $38, $1c, $05, $20        ;; 00:376c ????????
-    db   $f0, $d1, $c9                                 ;; 00:3774 ???
+
+;copy and roll-left-by-1 B bytes from HL in bank A to DE
```

Comment:
`copyAndRollBankAfromHLtoDE` is still not clear that it has a potential bug--that it depends on the high bits being zero, and that it is using `rlca` as a *shift*, not a *rotate*. (Where did the term "roll" come from?)

Reply:
I'm not seeing the potential bug - the function is working as intended and described, and as the caller I'm expecting the behavior I'm getting because I know the tiles I'm sending in have an empty bit on the left side. What am I missing?

As for roll vs rotate, that was my goof - sorry, fixed now!

Reply:
As long as you are clear that it works because of the empty bit on the left side, I'm happy with it.